packages/exporter-trace-otlp-http/src/OTLPExporterBase.ts

```diff
@@ -36,6 +37,7 @@ export abstract class OTLPExporterBase<
   protected _concurrencyLimit: number;
   protected _sendingPromises: Promise<unknown>[] = [];
   protected _shutdownOnce: BindOnceFuture<void>;
+  public readonly _timeoutMillis: number;
```

Comment:
public properties usually have no `_` prefix. I would also prefer to move it up a few lines to the other public properties.